workorders.lic

```diff
@@ -84,7 +84,12 @@ class WorkOrders
       if repair
         repair_items(info, tools)
       else
-        sew_items(info, item, quantity)
+	recipe = @recipes.find { |r| r['name'] == item }
```

Comment:
You have to look up the recipe again in both the sew and knit cases. Can `recipe` be passed in, instead of `item`? Or is this an indicator that the lookup and the if should be inside some intermediate function which contains a lot of the shared code between knitting and sewing?